vector_double.cpp

```diff
@@ -0,0 +1,207 @@
+#include "vector_double.hpp"
+
+
+template <typename T>
+Vector<T>::Vector()
+{
+    size_v = 0;
+    space  = 0;
+    elem   = nullptr;
+
+}
+
+
+// Copy Constructor
+template<class T>
+Vector<T>::Vector(const Vector<T>& b)
+{
+    space = b.space;
+    size_v = b.size_v;
+    elem = b.elem;
+
+}
+template<class T>
+Vector<T>::Vector(Vector<T>&& obj)
+{
+    space = obj.space;
+    size_v = obj.size_v;
+    elem = obj.elem;
```

Comment:
this is fine for a move constructor though

Reply:
it's fine, it just needs something plus a member initialization list
```suggestion
    elem = obj.elem;
    obj.elem = nullptr;
```
without it, we have a bug that delete's our vector's data when ``obj`` goes out of scope.